docs/source/60_building_apps.md

```diff
@@ -284,3 +284,327 @@ class MyCommandKryoRegistrator extends KryoRegistrator {
+
+    return jac.loadAlignments(filePath);
+  }
+}
+```
+
+With an `ADAMContext`, you can load:
+
+* Single reads as an `AlignmentRecordRDD`:
+  * From SAM/BAM/CRAM using `loadBam` (Scala only)
+  * Selected regions from an indexed BAM/CRAM using `loadIndexedBam` (Scala
+    only)
+  * From FASTQ using `loadFastq`, `loadPairedFastq`, and `loadUnpairedFastq`
+    (Scala only)
+  * From Parquet using `loadParquetAlignments` (Scala only)
+  * The `loadAlignments` method will load from any of the above formats, and
+    will autodetect the underlying format (Scala and Java, also supports loading
+    reads from FASTA)
+* Paired reads as a `FragmentRDD`:
+  * From interleaved FASTQ using `loadInterleavedFastqAsFragments` (Scala only)
+  * From Parquet using `loadParquetFragments` (Scala only)
+  * The `loadFragments` method will load from either of the above formats, as
+    well as SAM/BAM/CRAM, and will autodetect the underlying file format. If the
+    file is a SAM/BAM/CRAM file and the file is queryname sorted, the data will
+    be converted to fragments without performing a shuffle. (Scala and Java)
+* VCF lines as a `VariantContextRDD` from VCF/BCF1 using `loadVcf` (Scala only)
+* Selected lines from a tabix indexed VCF using `loadIndexedVcf` (Scala only)
+* Genotypes as a `GenotypeRDD`:
+  * From Parquet using `loadParquetGenotypes` (Scala only)
+  * From either Parquet or VCF/BCF1 using `loadGenotypes` (Scala and Java)
+* Variants as a `VariantRDD`:
+  * From Parquet using `loadParquetVariants` (Scala only)
+  * From either Parquet or VCF/BCF1 using `loadVariants` (Scala and Java)
+* Genomic features as a `FeatureRDD`:
+  * From BED using `loadBed` (Scala only)
+  * From GFF3 using `loadGff3` (Scala only)
+  * From GFF2/GTF using `loadGtf` (Scala only)
+  * From NarrowPeak using `loadNarrowPeak` (Scala only)
+  * From IntervalList using `loadIntervalList` (Scala only)
+  * From Parquet using `loadParquetFeatures` (Scala only)
+  * Autodetected from any of the above using `loadFeatures` (Scala and Java)
+* Fragmented contig sequence as a `NucleotideContigFragmentRDD`:
+  * From FASTA with `loadFasta` (Scala only)
+  * From Parquet with `loadParquetContigFragments` (Scala only)
+  * Autodetected from either of the above using `loadSequences` (Scala and Java)
+* Coverage data as a `CoverageRDD`:
+  * From Parquet using `loadParquetCoverage` (Scala only)
+  * From Parquet or any of the feature file formats using `loadCoverage` (Scala
+    only)
+* Contig sequence as a broadcastable `ReferenceFile` using `loadReferenceFile`,
+  which supports 2bit files, FASTA, and Parquet (Scala only)
+
+The `JavaADAMContext` class provides Java-friendly methods that are equivalent
+to the `ADAMContext` methods. Specifically, these methods use Java types, and do
+not make use of default parameters. In addition to the load/save methods
+described above, the `ADAMContext` adds the implicit methods needed for using
```

Comment:
Should we explicitly say that the Scala and Java labels in the above function calls indicate whether they can be used with JavaADAMContext/ADAMContext?